cmd/carapace/cmd/lazyinit/nushell.go

```diff
@@ -5,13 +5,26 @@ import "fmt"
 func Nushell(completers []string) string {
 	snippet := `%v%v
 
-let carapace_completer = {|spans| 
-  carapace $spans.0 nushell $spans | from json
+let carapace_completer = {|spans|
+  # if the current command is an alias, get it's expansion
+  let expanded_alias = (scope aliases | where name == $spans.0 | get -i 0 | get -i expansion)
```

Comment:
Looks fine, but isn't this something that should be done by nushell before passing the spans to the `external_completer`?
Other completers should be affected by this as well.

Reply:
Heho,

yes it should be.. but there exists a bug preventing extension. Thus, the given workaround should bei used. The code is taken from

https://www.nushell.sh/cookbook/external_completers.html#troubleshooting

Kind regards
Alexander

Reply:
Ah ok.